deps/build.jl

```diff
@@ -3,12 +3,12 @@ if PyCall.pyversion.major != 3 || PyCall.pyversion.minor != 7
     using Conda
     const Pkg = Base.require(Base.PkgId(Base.UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg"))
     Conda.add("python=3.7")
-    try
-        Conda.rm("numpy") #get around MKL problem
-    catch e 
-    end
-    Conda.add("nomkl")
-    Conda.add("numpy")
+    # try
+    #     Conda.rm("numpy") #get around MKL problem
+    # catch e 
+    # end
+    # Conda.add("nomkl")
+    # Conda.add("numpy")
```

Comment:
Are we sure we want to do this on main?

Reply:
Back then it was the only way I can get the CI test to pass for some reasons. I will remove them in the new push and see how it goes.